Extractor/UAClient.cs

```diff
@@ -283,6 +283,12 @@ private bool ShouldRetryException(Exception ex)
                     return Config.Source.Retries.FinalRetryStatusCodes.Contains(silentExc.InnerServiceException.StatusCode);
                 }
             }
+            else if (ex is AggregateException aex)
+            {
+                // Only retry aggregate exceptions if all inner exceptions should be retried...
```

Comment:
Only retry if all inner should be retried? If any should be retried, should we not retry that one?

Reply:
An aggregate exception technically means that multiple things have gone wrong... I guess we _could_ retry, I'm not 100% sure.